doc/ref_model/chapters/chapter01.md

```diff
@@ -53,28 +53,28 @@ This section defines the main terms used in this document; these deinitions are
+- **Virtual Network Function (VNF)**: a software implementation of a **Network Function**, capable of running on the **NFVi**.
+  - **VNF**s are built from one or more VNF Components (**VNFC**) and, in most cases,  the VNFC is hosted on a single VM or Container.
 - **Cloud-native (containerised) Network Function (CNF)**: **VNF** with a full adherence to cloud native principles, or a **VNF** that is transitioning to cloud native. 
   >_*Note:*_ It is a containerised **VNF** that is microservices-oriented, to increase agility and maintainability, and that can be dynamically orchestrated and managed to optimize resource utilization; the containers can be Linux, Docker or other similar container technology.
-- **vApp (VA)**: Virtual Application is more of a general term for software which can be loaded into a Virtual Machine. 
-  >_*Note:*_ a **VNF** is one type of vApp.
-- **Workload**: Workload refers to software running on top of **NFVI** resources such as **VMs** or **Container**s.
+- **Virtual Application (VA)**: is more of a general term for software which can be loaded into a Virtual Machine. 
+  >_*Note:*_ a **VNF** is one type of VA.
+- **Workload**: Workload refers to software running on top of compute resources such as **VMs** or **Container**s. Most relevant workload categories in context of NFVI are:
   - **Data Plane Workloads**: are related to packet handling in an end-to-end communication between applications. These tasks are expected to be very intensive in I/O operations and memory read/write operations.
   - **Control Plane Workloads**: are the task related to any other communication between NFs that is not directly related to the end-to-end data communication between applications. This category includes session management,routing or authentication.
   - **Storage Workloads**: are all tasks related to disk storage, from the non-intensive logging of a router, to more intensive read/write operations.
 - **Virtual Machine (VM)**: virtualised computation environment that behaves like a physical computer/server. 
-  >_*Note:*_ a **VM** consists of all of the components (processor (CPU), memory, storage, interfaces/ports, etc.) of a physical computer/server.
-- **VM Instance**: an abstract describtion of a VM that implies it's capabilities and resource usage. It has two parameters:
-  - **instance type**: The type of NFVI of which the VM is expected to run on. It directly maps to the combination of the underlying NFVI SW Profile and NFVI HW Profile.
-  - **compute flavour**: defines the compute, memory, and storage capacity of a virtual compute resource. 
+  >_*Note:*_ a **VM** consists of all of the components (processor (CPU), memory, storage, interfaces/ports, etc.) of a physical computer/server. It is created using Instance Type together with sizing information or Compute Flavour. 
+- **VM Instance**: is a VM
+- **Instance type**: defines the capabilities of the physical compute server that the virtual compute resource can run on.
```

Comment:
Hi @pgoyal01 can we please replace the sentence "physical compute server" in both instance type and compute flavour with "NFVI". Thanks